src/main/java/uk/ac/cam/cl/dtg/isaac/api/GameboardsFacade.java

```diff
@@ -108,19 +118,21 @@
      *            - for updating badge information.
      */
     @Inject
-    public GameboardsFacade(final PropertiesLoader properties, final ILogManager logManager,
-            final GameManager gameManager, final QuestionManager questionManager, final UserAccountManager userManager,
-            final UserAssociationManager associationManager,
-                            final UserBadgeManager userBadgeManager) {
+    public GameboardsFacade(final SegueContentFacade api, final PropertiesLoader properties, final ILogManager logManager,
+                            final GameManager gameManager, final QuestionManager questionManager,
+                            final UserAccountManager userManager, final UserAssociationManager associationManager,
+                            final UserBadgeManager userBadgeManager, @Named(CONTENT_INDEX) final String contentIndex) {
         super(properties, logManager);
 
+        this.api = api;
         this.userBadgeManager = userBadgeManager;
         this.gameManager = gameManager;
         this.questionManager = questionManager;
         this.userManager = userManager;
         this.associationManager = associationManager;
         String commaSeparatedIds = this.getProperties().getProperty(Constants.FASTTRACK_GAMEBOARD_WHITELIST);
         this.fastTrackGamebaordIds = new HashSet<>(Arrays.asList(commaSeparatedIds.split(",")));
+        this.contentIndex = contentIndex;
```

Comment:
After moving to using a content manager maybe we can remove the contentIndex, I don't think it should be necessary.

Reply:
It would still be necessary if you want to do what `SegueContentFacade` does (line 201).